src/components/ContentListToolbar/ContentListToolbar.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { ReactElement, useState } from 'react';
+import {
+    Select,
+    SelectOption,
+    SelectVariant,
+    TextInput,
+    Toolbar,
+    ToolbarContent, ToolbarFilter,
+    ToolbarGroup,
+    ToolbarItem
+} from '@patternfly/react-core';
+import { FilterIcon } from '@patternfly/react-icons';
+
+const mockArchitecturesVersions = {
+    'distribution_arches': [
+        {
+            'name': 'x86-64',
+            'label': 'x86_64'
+        },
+        {
+            'name': 'AArch64',
+            'label': 'aarch64'
+        }
+    ],
+    'distribution_versions': [
+        {
+            'name': 'RHEL 7',
+            'label': '7'
+        },
+        {
+            'name': 'RHEL 8',
+            'label': '8'
+        }
+    ]
+}
+
+type ContentListToolbarProps = {
+    setSearchQuery: any,
+}
+
+const ContentListToolbar = ({setSearchQuery}: ContentListToolbarProps) => {
+    const versions = {};
+    const arches = {};
+    mockArchitecturesVersions['distribution_arches'].map(arch => arches[arch.name] = arch.label);
+    mockArchitecturesVersions['distribution_versions'].map(version => versions[version.name] = version.label);
+
+    const filterTypes = ['Name/URL', 'Version', 'Architecture'];
+    const [selectedFilters, setSelectedFilters] = useState({
+        filterType: 'Name/URL',
+        versions: [],
+        architectures: [],
+    });
+    const [filterIsOpen, setFilterIsOpen] = useState(false);
+    const [versionIsOpen, setVersionIsOpen] = useState(false);
+    const [archIsOpen, setArchIsOpen] = useState(false);
+
+    const selectFrom = (options: Array<string>) => {
```

Comment:
where possible, for this and all other simple functions, use a direct return. 
someFunc = (vars) => vars.dothings()